design/one-pager-definition-of-done.md

```diff
@@ -0,0 +1,40 @@
+# Definition of done
+
+* Owner: Daniel Suskin (@suskin)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Introduction
+
+*Why does this document exist?*
+
+The goal of this document is to answer the questions:
+
+* As a developer, when should I consider a chunk of work to be complete?
+* As a reviewer, what should I be looking for to verify that a chunk of work is
+  complete?
+
+## Definition of work
+
+Work means a contribution to the project, typically code.
+
+## Definition of done
+
+A chunk of work is considered done when the following criteria have been met:
+
+* All code is written
+* Code is reviewed and approved
+* New behavior is documented in the documentation that will go onto the
```

Comment:
Do we want to call out any required updates to `examples/` as well? They are not specifically used in the Crossplane website, but I think we should desire for them to always be up to date.

Reply:
This is a good point . . . let me think about how to include this as well

Reply:
I have added a line about this in the new section of things which **should** be done